Added a sentence on the impact of cloud resources (#818)
digital-asset/docs.daml.com: docs/2.10.0/docs/canton/usermanual/persistence.rst

Most Canton indexes are contract-id based, which means that the index lookups are randomly distributed. Solid state drives with
high throughput perform much better than spinning disks for this purpose.

.. _shared_env_performance:

Predictability of Shared Environments
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> The throughput and latency of a Canton node is highly dependent on the performance of the database. 

Reviewer: ```suggestion
The throughput and latency of a Canton node depend on the performance of the database. 
```

Author: fixed

> Sharing hardware or software is an efficient method to save cost and better utilize available resources, but comes 

Reviewer: ```suggestion
Sharing hardware or software saves cost and better utilizes available resources, but it comes 
```

Author: fixed

> with some drawbacks: If the database is operated in a shared environment such as the Cloud, where other applications are either using the same 

Reviewer: ```suggestion
with some drawbacks: If the database is operated in a shared environment such as the Cloud, where other applications are using the same 
```

Author: fixed

> database or are operated on the same hardware, the performance of the Canton node will vary due to 

Reviewer: ```suggestion
database or are operated on the same hardware, the performance of the Canton node varies due to 
```

Author: fixed

> contention on shared resources. This is a natural effect of shared environments and cannot be fundamentally 
> avoided, but can be difficult to diagnose as a user of the shared environment due to lack of visibility into 
> the other applications.

Reviewer: ```suggestion
contention on shared resources. This is a natural effect of shared environments and cannot be entirely 
avoided. It can be difficult to diagnose as a user of the shared environment due to a lack of visibility into 
the other applications.
```

Author: fixed
